api_specifications/open-api-0.1.0.yaml

```
@@ -0,0 +1,218 @@
+  title: All Public/Open APIs supported by the platform
+  description: This API specification defines the various APIs that are open to external world. Anyone can call these. These are unauthenticated APIs. These need to managed via rate limiting settings at the gateway level to limit access
+  version: 0.1.0
+tags:
+  - name: Open APIs
+paths:
+  /openapi/{tenantID}/case/cnr/{cnrNumber}:
+    get:
+      tags:
+        - Open APIs
+      summary: get case summary by cnr number
+      description: The API is used to get case summary details using the case CNR Number within a specific tenant. 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)